sfdx-project.json

```diff
@@ -0,0 +1,11 @@
+{
+    "packageDirectories": [
+        {
+            "path": "sfdx-source/apex-mocks",
```

Comment:
I'm not proposing a change here .. just asking a question: How did you arrive at the sfdx-source top level directory name? we usually use 'force-app' for our local application code and we pull in 'fflib-apex-common' and 'fflib-apex-mocks' as top level siblings to the 'force-app' dir

Reply:
@daveespo -- this is the basic format that comes from the "Falcon Template" approach of DX project setup.  It allows you to add the other source directories related to the project like `sfdx-source/unpackaged` and `sfdx-source/untracked`.  I have found it to be a great setup in the projects that I have worked with.  I am certainly open to other suggestions though.